x-conduit/src/X/Data/Conduit.hs

```diff
@@ -0,0 +1,10 @@
+module X.Data.Conduit (
+    yieldM
+  ) where
+
+
+import           Control.Monad.Trans.Class (lift)
+import           Data.Conduit (yield, ConduitM(..))
+
+yieldM :: Monad m => m o -> ConduitM i o m ()
+yieldM mo = lift mo >>= yield
```

Comment:
Not sure if you want to add a comment about the two `yieldM` versions in conduit?